koku/reporting/provider/aws/models.py

```diff
@@ -761,3 +761,17 @@ class Meta:
     markup_cost = models.DecimalField(max_digits=24, decimal_places=9, null=True)
 
     currency_code = models.CharField(max_length=10)
+
+
+class AWSOrganizationalUnit(models.Model):
+    """The alias table for AWS Organizational Unit."""
+
+    org_unit_name = models.CharField(max_length=250, null=False, unique=False)
+
+    org_unit_id = models.CharField(max_length=50, null=False, unique=False)
+
+    org_unit_path = models.TextField(null=False, unique=False)
+
+    account_id = models.CharField(max_length=50, null=True, unique=False)
+
+    created_timestamp = models.DateField(null=True)
```

Comment:
Optional: You can have the timestamp auto set using https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.DateField.auto_now_add